src/api/db/models/Project.js

```diff
@@ -488,7 +488,9 @@ export class ProjectModel {
       const label = (project.labels || []).filter((p) => { return p._id.toString() === input._id.toString(); })[0];
       if (!label) throw new DeleteLabelError('Label not found on project');
 
-      const count = await ImageModel.countImagesByLabel([input._id], context);
+      const count = await ImageModel.countImagesByLabel([input._id], {
```

Comment:
@ingalls - guessing this is a relic from when you tried to limit the count aggregation? At any rate it doesn't look like that `limit` option is used or parsed on the `countImagesByLabel` function.